dash/include/dash/atomic/Operation.h

```diff
@@ -31,15 +31,15 @@ namespace atomic {
  * Get the value of the shared atomic.
  */
 template<typename T>
-T load(const dash::GlobRef<dash::Atomic<T>> & ref){
+T load(const dash::GlobRef<dash::Atomic<T>>& ref){
```

Comment:
This is a consequence of the removed `const` in `GlobRef`. There is a documentation entry in GlobRef explaining why the `const` is correct there. There is also a proposal (don't have it at my finger tips currently) which explains this in more detail.

Reply:
Ok, that convinces me.

Reply:
No. There is no such thing as a "const reference", there only is "reference to const".
This is why I implemented `GlobRef` to return `const T` / `const T &` when accessing a `const GlobRef<T>`. Global references are references, not iterators.

Reply:
Obviously, the problem is that the const in GlobRef<const T> is simply lost when accessing the data through an external interface.

So what are you suggesting? To disable member functions altering the value referenced by GlobRef<T> for T=const X?